components/esp_modem/port/linux/esp_system_protocols_linux/esp_err_to_name.c

```diff
@@ -0,0 +1,15 @@
+/*
+ * x.c
+ *
+ *  Created on: 06.07.2022
+ *      Author: franz
```

Comment:
Note that you can add your own copyright notice, we'll default to the Apache licence.